sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestDocumentation.java

```diff
@@ -83,9 +83,8 @@ public void testFiveMinuteSqlObjectExample() {
     @Test
     public void testObtainHandleViaOpen() {
         assertThatCode(() -> {
-            try (Handle h = h2Extension.openHandle()) {
-                // nop
-            }
+            Handle h = h2Extension.getJdbi().open();
```

Comment:
this causes a warning in intellij (Using AutoCloseable Handle without t-w-r...)

Reply:
`EmptyBlock` check is gone and the this change reverted